docs/api-reference/sleep/get-stream.mdx

```diff
@@ -104,37 +104,47 @@ data = await client.Sleep.getStream(sleep_id=*)
     {
       "id": 0,
       "timestamp": "2022-08-04T12:42:45.722168+00:00",
+      "start": "2022-08-04T12:42:45.722168+00:00",
+      "end": "2022-08-04T12:43:15.722179+00:00",
       "value": 1,
-      "type": "vital_hypnogram",
-      "unit": "vital_hypnogram"
+      "type": "None",
```

Comment:
These should be JSON null 🤔

Reply:
aaah, will fix

Reply:
https://github.com/tryVital/docs/pull/139